### PR TITLE
Cleanup some copyright notices

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

- Adds some missing copyright notices
- Updates a handful of notices that still report Pixie Brix, LLC
- Always add two line breaks after the notice
- Drop `!` from start from some files (not useful in this context)

## Future work

I think that there are like 50 more files without notice, but that requires _some_ review if I automate it, so I skipped it for now.

### CODE REVIEW

File: selenium/environment.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
+ * GNU Affero General Public License for more details.
  *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (fregante) [inline selenium/environment.js:15]: All notices use http:// 😬

I tried doing a search and replace to use https, but that added 1300 files to this diff so I didn't push. Would you like this change?

- [ ] Yes
- [ ] No
- [ ] Maybe

**Comment by twschiller** [comment]
Thanks, @fregante. IntelliJ supports file copyright notice management. Instead of modifying these notices manually, we need to update the IntelliJ template and use IntelliJ to reapply the notices

File: selenium/environment.js (hunk 1 above)
Comment (twschiller) [reply selenium/environment.js:15]: OK, looks like they had updated the link in their copyright disclaimer: https://www.gnu.org/licenses/agpl-3.0.en.html

See my other comment re: using IntelliJ to manage the copyright notices

**Comment by fregante** [comment]
> IntelliJ supports file copyright notice management 

Indeed, that works too.


> someone else from the team can update and apply

Yeah my license expired and I haven't used it often enough to justify a new one, I think 😅 

Either way probably not worth holding up this PR and risk merge conflicts. Further update can come on January 1st 2023 😁

**Review by twschiller — APPROVED** [review]

**Pull request merged**